src/app/Recordings/ActiveRecordingsTable.tsx

```diff
@@ -462,115 +472,68 @@ export const ActiveRecordingsTable: React.FunctionComponent<ActiveRecordingsTabl
     );
   };
 
-  const toggleExpanded = (id) => {
-    const idx = expandedRows.indexOf(id);
-    setExpandedRows(expandedRows => idx >= 0 ? [...expandedRows.slice(0, idx), ...expandedRows.slice(idx + 1, expandedRows.length)] : [...expandedRows, id]);
-  };
-
-  const handleDeleteButton = React.useCallback(() => {
-    if (context.settings.deletionDialogsEnabledFor(DeleteWarningType.DeleteActiveRecordings)) {
-      setWarningModalOpen(true);
-    }
-    else {
-      handleDeleteRecordings();
-    }
-  }, [context, context.settings, setWarningModalOpen, handleDeleteRecordings]);
+  const toggleExpanded = React.useCallback((id: string) => {
+    setExpandedRows(expandedRows => {
+      const idx = expandedRows.indexOf(id);
+      return idx >= 0 ? [...expandedRows.slice(0, idx), ...expandedRows.slice(idx + 1, expandedRows.length)] : [...expandedRows, id]
+    });
+  }, [expandedRows, setExpandedRows]);
 
-  const handleWarningModalClose = React.useCallback(() => {
-    setWarningModalOpen(false);
-  }, [setWarningModalOpen]);
+  const isDeleteDialogEnabled = React.useMemo(
```

Comment:
Since the deletion warning thing is in the toolbar now, this check might be better off in there as well.

Reply:
Right makes sense :D I will fix that!